app/utils.ts

```diff
@@ -0,0 +1,13 @@
+// tslint:disable-next-line
+import { BigNumber } from "bignumber.js";
+import { commitmentStartDate } from "./config";
+import { asEtherNumber } from "./web3/utils";
+
+export async function loadIcoParamsFromEnviroment() {
```

Comment:
Why is this not part of the config file?

Reply:
Actually good point